qutip/visualization.py

```diff
@@ -1206,8 +1203,7 @@ def plot_expectation_values(results, ylabels=[], title=None, show_legend=False,
             axes[e_idx, 0].plot(result.times, e,
                                 label="%s [%d]" % (result.solver, e_idx))
 
-    if title:
-        axes[0, 0].set_title(title)
+    fig.suptitle(title)
```

Comment:
What does this change do?

Reply:
If you have many other axes, the title is not assigned to `axes[0, 0]`, but to the whole figure.

Reply:
```suggestion
    if title:
        fig.suptitle(title)
```